src/foam/mlang/mlang.js

```diff
@@ -533,6 +538,11 @@ foam.CLASS({
       type: 'String',
       javaCode: 'return " 1 <> 1 ";',
       code: function() { return "1 <> 1"; }
+    },
+    {
+      name: 'partialEval',
+      code: function() { return this },
+      javaCode: 'return foam.mlang.MLang.FALSE;'
```

Comment:
Isn't this what already happens?

Reply:
No, the singleton axiom doesn't work on java.

Reply:
We can fix this in the future.